common/autoware_auto_common/include/common/filesystem.hpp

```diff
@@ -37,8 +37,8 @@
 // #endif
 //
 //---------------------------------------------------------------------------------------
-#ifndef GHC_FILESYSTEM_H
-#define GHC_FILESYSTEM_H
+#ifndef COMMON__FILESYSTEM_HPP_
+#define COMMON__FILESYSTEM_HPP_
```

Comment:
@isamu-takagi How do we handle third-party header files? Ignoring them in `.pre-commit-config.yaml`?

Reply:
Hmm. How about this?
```
- id: ros-include-guard
  exclude: common/autoware_auto_common/include/common/filesystem\.hpp
```

Reply:
It's fine.
IMO it's also okay if we overwrite the macros in third-party headers because there are few files.
Which do you prefer?

Reply:
It's easier if there is no exclude pattern. Let's do that.

Reply:
BTW, actually I think this file can be removed because Galactic supports C++17.

Reply:
#440